RELEASING.md

```diff
@@ -56,42 +56,58 @@ Please see the [Main Readme](README.md) for details on building protobuf.
 
 Tagging the Release
 ----------------------
-The first step in the release process is to create a release branch and then
-from it, create a tag for the release. Our release branches follow the naming
+The first step in the release process is to create a release branch, bump
+versions, and create a tag for the release. Our release branches follow the naming
 convention of `v<major>.<minor>.x`, while the tags include the patch version
 `v<major>.<minor>.<patch>`. For example, the same branch `v0.7.x`
 would be used to create all `v0.7` tags (e.g. `v0.7.0`, `v0.7.1`).
 
-1. Create the release branch:
+1. Create the release branch and push it to GitHub:
 
    ```bash
    $ git checkout -b v<major>.<minor>.x master
+   $ git push upstream v<major>.<minor>.x
    ```
-2. Next, increment the version in `build.gradle` in `master` to the next
-   minor snapshot (e.g. ``0.8.0-SNAPSHOT``).
-3. In the release branch, change the `build.gradle` to the next release version
-   (e.g. `0.7.0`)
-4. Push the release branch to github
+2. For `master`, change `build.gradle` and
+   `android-interop-testing/app/build.gradle` to the next minor snapshot (e.g.
+   ``0.8.0-SNAPSHOT``).
 
    ```bash
-   $ git push upstream v<major>.<minor>.x
+   $ git checkout -b bump-version master
+   # Change version to next minor (and keep -SNAPSHOT)
```

Comment:
Updating the version in the files is a major step, but it can be easily ignored here as its a grayed-out comment.
Maybe change it to a `sed` command that actually does the work?